### ISSUE TITLE
Errors when following "Getting Started" guide

### ISSUE THREAD

**Issue by redstar504** [hashicorp/vagrant#2258]
Hello,

I am running VirtualBox 4.2.18 and Vagrant 1.3.3 on Windows 7 64-bit.

I have been hung up simply trying to get the precise32 box from the [getting started guide](http://docs.vagrantup.com/v2/getting-started/index.html) working.  I am experiencing errors while running the vagrant up command.

My problem is that the VM boots successfully but I am receiving boot_timeout error messages.  I have tried increasing the config.vm.boot_timeout to various levels without luck.  From the log this appears to happen due to Vagrant being unable to connect to SSH on 127.0.0.1, however if I use PuTTy or suspend vagrant up and run vagrant ssh manually,  I can successfully connect to the VM's SSH without issue (127.0.0.1:2222).

Please see my [Vagrantfile](https://gist.github.com/redstar504/5578bde596c35f69a58f),  and [debug level log](https://gist.github.com/redstar504/6666563).

Basically, I would just like to be able to run the "vagrant up" command from the context of the getting started guide without experiencing any errors.   Any help appreciated! :+1:
